src/otx/algo/segmentation/backbones/litehrnet.py

```diff
@@ -1191,7 +1187,7 @@ def forward(self, x: torch.Tensor) -> list[torch.Tensor]:
         return out
 
 
-class LiteHRNet(BaseModule):
+class NNLiteHRNet(nn.Module):
```

Comment:
Just a naming suggestion, how about using `LiteHRNetModule` instead of `NNLiteHRNet`?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3863